ServantPersistent/src/Models.hs

```diff
@@ -29,8 +29,7 @@ DBTenant json
     status TenantStatus
     createdAt UTCTime
     updatedAt UTCTime
-    UniqueTenant name
-    UinqueBackofficeDomain backofficeDomain
+    UniqueBackofficeDomain backofficeDomain
```

Comment:
@saurabhnanda Persistent creates constructors that represent the uniqueness constraints.

For example, if you want to get a tenant from a unique backoffice domain, you can simply use

    getBy (UniqueBackofficeDomain "example.com")

instead of using something like a `select`.